tests/test.py

```diff
@@ -2675,6 +2693,11 @@ def get_poison_record_data(self):
         return get_poison_record_with_acrablock()
 
 
+class TestShutdownPoisonRecordWithZoneAcraBlockWithCachedKeystore(KeystoreCacheOnStartMixin, TestShutdownPoisonRecordWithZoneAcraBlock):
```

Comment:
why you decided to use this mixin here?)

Reply:
Decide to rewrite this to have two separate classes which test Enabled/Disabled `cache_on_start` functionality